### DISCUSSION TITLE
How do you update your suggestion option based on the data from an api ?

### DISCUSSION THREAD

**Discussion by RavinRau** [suren-atoyan/monaco-react#373] [Q&A]
Hi, I am able to get the option from the API but it will only work whenever I do ctrl + space to trigger the new options. I want it to update automatically without the need to ctrl + space every time the API call is completed. I try searching the whole internet for a solution but can't find one so I am asking my question here. Hopefully to hear an answer soon.

**Comment by suren-atoyan** [comment]
hi @RavinRau sorry for the late answer. Could you please share a reproducible code snippet? Do you use `provideCompletionItems`?
